README.md

```diff
@@ -88,7 +88,7 @@ available for you to run this library.
 
 We use a configuration file called _wskprop_ to specify all the parameters necessary for this Go client library to access the OpenWhisk services. Make sure you create or edit the file _~/.wskprops_, and add the mandatory parameters APIHOST, APIVERSION, NAMESPACE and AUTH.
 
-- The parameter `APIHOST` is the OpenWhisk API hostname (for example, openwhisk.ng.bluemix.net, 172.17.0.1, and so on).
+- The parameter `APIHOST` is the OpenWhisk API hostname (for example, us-east.functions.cloud.ibm.com, 172.17.0.1, and so on).
```

Comment:
Could you add some text explaining that this APIHOST endpoint is a sample and you can substitute with that of any OW deployment?  It would be great if we could have a "if you run OW locally your APIHOST might look like: xxx).

Reply:
For example, the "tools/dev" README indicates the default Controller port to be used: https://github.com/apache/openwhisk/blob/master/tools/dev/README.md

```~/.wskprops must be updated with APIHOST=http://localhost:10001 so that the wsk CLI communicates directly with the controller.```

Reply:
and the "standalone server" starts on a diff. host:port:
https://github.com/apache/openwhisk/tree/master/core/standalone

It would be good to link/and explain these 2 common setting variants for APIHOST

Reply:
Update example for local stand-alone and cloud functions